cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -99,7 +99,8 @@ private static int _main() throws UnknownHostException {
         final long      healthCheckIntervalSec  = Long.parseLong(       Env.option("CASSANDRA_HEALTH_CHECK_INTERVAL_SECONDS").or("60"));
         final long      bootstrapGraceTimeSec   = Long.parseLong(       Env.option("CASSANDRA_BOOTSTRAP_GRACE_TIME_SECONDS").or("120"));
         final String    cassandraVersion        =                       "2.1.2"; // TODO Env.option("CASSANDRA_VERSION").or("2.1.2");
-        final String    frameworkName           = frameworkName(        Env.option("CASSANDRA_CLUSTER_NAME"));
+        final String    clusterName             = frameworkName(        Env.option("CASSANDRA_CLUSTER_NAME"));
+        final String    frameworkName           = frameworkName(        Env.option("FRAMEWORK_NAME").or(Env.option("CASSANDRA_CLUSTER_NAME")));
```

Comment:
We don't need the framework name and the zk url manipulation. The zk URL that is provided allows for nesting of nodes if the user wants to do that. See #68 where I added several tests to illustrate the supported urls and formats.